olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -515,6 +517,14 @@ spec:
                     volumeMounts:
                       - mountPath: /cli-artifacts
                         name: cli-artifacts
+                    securityContext:
+                      allowPrivilegeEscalation: false
+                      runAsNonRoot: true
+                      capabilities:
+                        drop:
+                          - ALL
+                      seccompProfile:
+                        type: RuntimeDefault
```

Comment:
Should we see if we can land those on upstream? Or directly on midstream first ?